README.md

```diff
@@ -79,6 +79,20 @@ The databases we see most frequently used in the community are:
 
 Please visit [Installation guideline](docs/installation.md) on how to install Amundsen.
 
+## Service version guide
```

Comment:
about a migration guide or faq doc?

Reply:
I like the idea of a dedicated dos because over time we will keep adding to it. This section could stay here for high visibility, but just link to the separate doc. Fine to merge though.

Reply:
yeah, once we have a faq doc, we could copy some of the repetitive slack q & a into that doc as well.

Reply:
+1 on separate doc. Will update!